platformio.ini

```diff
@@ -30,6 +30,12 @@ build_src_filter = +<*> -<.git/> -<.svn/>
 ; for version requirements see https://docs.platformio.org/en/stable/core/userguide/pkg/cmd_install.html#cmd-pkg-install-requirements
 lib_deps = 
   adafruit/Adafruit SSD1306@~2.5.3
+  lvgl@~8.3.7
+build_flags =
+  -DLV_KCONFIG_IGNORE         ; lvgl: skip some default values that are used for ESP-IDF
+  -DLV_CONF_INCLUDE_SIMPLE    ; lvgl: use a config file located in source directory
+  -Wno-discarded-qualifiers   ; removes warings about discarding qualifiers, but creates "notes" that it has no effect on C++ compiler
```

Comment:
I suggest to leave the warning (remove suppression). Because

- it gives more warnings/notes on C++ source files (that it is not applicable) than actual warnings in C-files
- in general it is a good idea to have warnings enabled
- maybe/hopefully this issue will be resolved in the library (see [current discussion in forum](https://forum.lvgl.io/t/compiler-warnings-assignment-discards-const-qualifier-from-pointer-target-type/6502/9?u=hebster) and [outdated pull request](https://github.com/lvgl/lvgl/pull/2508))

```suggestion
```